build/Gruntfile.coffee

```diff
@@ -77,6 +80,11 @@ module.exports = (grunt) ->
       dest: appDir
       ext: '.js'
 
+  babelConfig =
+    options: {}
```

Comment:
Should this include any of the options in the `defaultOptions` object from `src/babel.coffee`? Not sure what the default options are for source maps.